htdocs/product/class/product.class.php

```diff
@@ -4250,8 +4251,10 @@ public function get_nb_mos($socid, $mode, $filteronproducttype = -1, $year = 0,
 	 * @param  int $incdec  1=Increase/decrease stock of child when parent stock increase/decrease
 	 * @return int                < 0 if KO, > 0 if OK
```

Comment:
@altairis-benjamin do not forget to add doc comment for new parameters

Reply:
You're perfectly right, doing it right now and will do a new commit. Thanks for you attention.